frontend/coprs_frontend/coprs/logic/builds_logic.py

```diff
@@ -460,8 +460,8 @@ def get_build_task(cls, task_id):
         except ValueError:
             raise MalformedArgumentException("Invalid task_id {}".format(task_id))
 
-        build_chroot = BuildChrootsLogic.get_by_build_id_and_name(build_id, chroot_name)
         try:
+            build_chroot = BuildChrootsLogic.get_by_build_id_and_name(build_id, chroot_name)
```

Comment:
This is a valid code movement.

Reply:
true, that's maybe the only thing I would want to accept right now from this PR